src/osemosys2iamc/resultify.py

```diff
@@ -26,9 +26,40 @@
 import matplotlib.dates as mdates
 
 
-def read_file(filename) -> pd.DataFrame:
+def read_file(in_path: str, osemosys_param: str, region_name_option: str) -> pd.DataFrame:
```

Comment:
This function reads results from a different result file format to that used by other models. If this is a requirement, I suggest first writing a script which converts all the results to the standard csv format defined by otoole, and then using the existing code.